src/controllers/user/routes.ts

```diff
@@ -17,5 +18,9 @@ userRouter.route('/')
 .put(authmiddleware('getUser','read'),validationHandler ( config) ,  userControler.update)
 .delete(authmiddleware('getUser','read'),validationHandler ( config) ,userControler.delete)
```

Comment:
1. In validationHandler() you have passed the full config object instead of passing config related to that route. For example config.create for creating, config.delete for delete etc.
2. You have imported validation config from trainee folder it should be from users as we have different validation defined for user.